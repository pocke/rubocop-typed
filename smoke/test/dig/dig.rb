a = {a: { b: 1 } }
p a[:a][:b]

b = {a: [{b: 1}]}
p b[:a][0][:b]

c = nil
c[:a][:b]
