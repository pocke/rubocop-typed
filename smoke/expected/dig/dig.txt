C:  2:  3: Typed/Dig: Use dig instead of [].
C:  5:  3: Typed/Dig: Use dig instead of [].
